README.md

```diff
@@ -2,12 +2,12 @@
 <img src="https://i.imgur.com/4nkFjdv.png" height="80px">
 <h1>ArchiveBox<br/><sub>The open-source self-hosted web archive.</sub></h1>
 
-▶️ <a href="https://github.com/pirate/ArchiveBox/wiki/Quickstart">Quickstart</a> | 
-<a href="https://archivebox.zervice.io/">Demo</a> | 
-<a href="https://github.com/pirate/ArchiveBox">Github</a> | 
-<a href="https://github.com/pirate/ArchiveBox/wiki">Documentation</a> | 
-<a href="#background--motivation">Info & Motivation</a> | 
-<a href="https://github.com/pirate/ArchiveBox/wiki/Web-Archiving-Community">Community</a> | 
+▶️ <a href="https://github.com/pirate/ArchiveBox/wiki/Quickstart">Quickstart</a> |
+<a href="https://archivebox.zervice.io/">Demo</a> |
+<a href="https://github.com/pirate/ArchiveBox">Github</a> |
+<a href="https://github.com/pirate/ArchiveBox/wiki">Documentation</a> |
+<a href="#background--motivation">Info & Motivation</a> |
+<a href="https://github.com/pirate/ArchiveBox/wiki/Web-Archiving-Community">Community</a> |
```

Comment:
careful, this whitespace is significant in markdown

Reply:
argh, vscode messing with it. I will revert it.

Reply:
> careful, this whitespace is significant in markdown

Is this for the rendering in some other place than github? https://github.com/pirate/ArchiveBox/tree/node_config looks the same: https://github.com/pirate/ArchiveBox
I can revert it, just wondering what it breaks